server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/EstablishAuditBoard.java

```diff
@@ -104,12 +104,14 @@ public String endpoint(final Request the_request,
           ok(the_response, "audit board for county " + county +  
                            " set to " + audit_board);
         }
+      } else {
+        invariantViolation(the_response, "Invalid audit board membership");
       }
     } catch (final PersistenceException e) {
       serverError(the_response, "unable to set audit board: " + e);
 
     } catch (final JsonSyntaxException e) {
-      invariantViolation(the_response, "Invalid audit board specified");
+      badDataContents(the_response, "Invalid audit board data");
```

Comment:
No, I think it's actually bad contents - unparsable JSON, as opposed to an audit board that violates an invariant.